README.rst

```diff
@@ -34,6 +34,16 @@ command::
 
   $ celery-prometheus-exporter
   Starting HTTPD on 0.0.0.0:8888
+  
+  
+Building
+==========
+Build the container with
+`docker build -f Dockerfile-celery3 -t spothero/redis-celery-exporter:$VERSION --build-arg BUILD_VERSION=$VERSION .`
+replacing `$VERSION` with the version you're building
+
+Push container with
+`docker push spothero/redis-celery-exporter:$VERSION`
```

Comment:
Should this come before/after the "How to use" section? Or is meant to be in the middle?

Reply:
Fixed this, don't know how it got there like that.